code/game/objects/items/weapons/tanks/tanks.dm

```diff
@@ -21,8 +21,25 @@
 	var/volume = 70
 
 /obj/item/weapon/tank/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
 	user.visible_message("<span class='suicide'>[user] is putting the [src]'s valve to their lips! I don't think they're gonna stop!</span>")
 	playsound(loc, 'sound/effects/spray.ogg', 10, 1, -3)
+	if (H && !qdeleted(H))
+		for(var/obj/item/W in (H.contents))
+			H.unEquip(W)
+			if (H.client)
+				H.client.screen -= W
+			if (W)
+				W.loc = H.loc
+				W.dropped(H)
+				if(prob(50))
+					step(W, pick(alldirs))
+		H.hair_style = "Bald"
+		H.update_hair()
+		H.blood_max = 5
+		gibs(H.loc, H.viruses, H.dna)
+		H.adjustBruteLoss(9001) //to make the body super-bloody

```

Comment:
It's not a new meme; it's a re-introduction of #11700, but fixing the issues referenced in #12159 and #12224